### ISSUE TITLE
does it require persistence?

### ISSUE THREAD

**Issue by ORESoftware** [bahmutov/cache-require-paths#14]
I don't understand why this requires persistence to a .json file? Is there a way to do this in memory? Or maybe it's because on the second run it can lookup in the .json file? An explanation of how it works in the readme would be nice.

**Comment by ivan** [comment]
"The first time the app loads, a cache of resolved file paths will be saved to .cache-require-paths.json in the current directory. Every application startup after that will reuse this filename cache to avoid "hunting" for the right filename."

**Comment by ORESoftware** [comment]
yeah I saw that afterwards haha ok. The only remaining question i have - it seems like it only works with dependencies in node_modules, not other files. I suppose there might be 100s of deps in node_modules though.
